applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.c

```diff
@@ -12,12 +12,32 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_DECLARE(ble, CONFIG_BLE_LOG_LEVEL);
 
-enum ble_hci_vs_max_tx_power {
-	BLE_HCI_VSC_MAX_TX_PWR_0dBm = 0,
-	BLE_HCI_VSC_MAX_TX_PWR_3dBm = 3,
-};
+int ble_hci_vsc_set_fem_pin(struct ble_hci_vs_cp_set_fem_pin *fem_pin)
+{
+	int ret;
+	struct ble_hci_vs_cp_set_fem_pin *cp;
+	struct net_buf *buf = NULL;
+
+	buf = bt_hci_cmd_create(HCI_OPCODE_VS_CONFIG_FEM_PIN, sizeof(*cp));
+	if (!buf) {
+		LOG_ERR("Unable to allocate command buffer");
+		return -ENOMEM;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->mode = fem_pin->mode;
+	cp->txen = fem_pin->txen;
+	cp->rxen = fem_pin->rxen;
+	cp->antsel = fem_pin->antsel;
+	cp->pdn = fem_pin->pdn;
+	cp->csn = fem_pin->csn;
+
+	ret = bt_hci_cmd_send(HCI_OPCODE_VS_CONFIG_FEM_PIN, buf);
+
+	return ret;
+}
 
-int ble_hci_vsc_set_radio_high_pwr_mode(bool high_power_mode)
+int ble_hci_vsc_set_radio_high_pwr_mode(enum ble_hci_vs_max_tx_power max_tx_power)
```

Comment:
Why "max"? Is it not the "usual" tx power?

Reply:
Will not remove max after all. This is a value we give to the FEM to let it know that we might potentially ask for max power, but even if we set this to 20dBm we could still set the actual tx power to 10dBm